src/sentry/models/integrations/organization_integration.py

```diff
@@ -56,3 +56,32 @@ def delete(self, *args, **kwds):
             for outbox in self.outboxes_for_update():
                 outbox.save()
             super().delete(*args, **kwds)
+
+    @staticmethod
+    def services_in(config: Mapping[str, Any]) -> List[Any]:
+        return config.get("pagerduty_services", [])
+
+    def add_pagerduty_service(self, integration_key: str, service_name: str) -> Any:
```

Comment:
Typing will swap from `Any` to a typed dict in the follow up PR but this is circular dependency on the way through getsentry, so it has to be an Any until after the getsentry part lands.